### DISCUSSION TITLE
M5Stack　LEDテープが光らないことについて

### DISCUSSION THREAD

**Discussion by CharlieW-JP** [1ft-seabass/ma-hackathon-qa-and-knowledge-202210#25] [Q&A] [answered]
加速度センサーとLEDテープをM5Stackに繋げて、加速度センサーの値が変わると変数の値を加算していきます。
変数の値によってLEDの色を変える仕組みです。
コードは下記の画像の通りです。
LEDが光らないことがなぜでしょうか。

<img width="743" alt="スクリーンショット 2022-11-07 18 08 16" src="https://user-images.githubusercontent.com/64363028/200271140-6dc3b8a4-4412-4e2d-9c13-450efea30340.png">

**Comment by 1ft-seabass** [comment, marked as answer]
おおお。UIFlowでやってるのですね？ややサポートが難しいのですが、内容みてみます！

@CharlieW-JP

**Reply by 1ft-seabass** [reply]
@CharlieW-JP こちらについては、

- LED を光らせる処理が loop 実行中に連続するとマズそうなので loop 処理に delay を入れる

という解決方法でしたね～。おつかれさまでしたー。
